CMakeLists.txt

```diff
@@ -46,15 +46,24 @@ option(ENABLE_MKLGPU_BACKEND "" ON)
 if(ENABLE_MKLCPU_BACKEND)
   option(ENABLE_MKLCPU_THREAD_TBB "" ON)
 endif()
-option(ENABLE_CUBLAS_BACKEND "" OFF)
-
-option(ENABLE_CUSOLVER_BACKEND "" OFF)
 
+# blas
+option(ENABLE_CUBLAS_BACKEND "" OFF)
 option(ENABLE_ROCBLAS_BACKEND "" OFF)
+option(ENABLE_NETLIB_BACKEND "" OFF)
+
+# rand
 option(ENABLE_CURAND_BACKEND "" OFF)
 option(ENABLE_ROCRAND_BACKEND "" OFF)
+
+# lapack
+option(ENABLE_CUSOLVER_BACKEND "" OFF)
 option(ENABLE_ROCSOLVER_BACKEND "" OFF)
-option(ENABLE_NETLIB_BACKEND "" OFF)
+
+# dft
+option(ENABLE_CUFFT_BACKEND "Enable the cuFFT backend for DFT" OFF)
```

Comment:
That is help text. It will appear in CMakeCache.txt (in the build folder) or when using a configuration tool like `ccmake`. e.g. from my CMakeCache.txt
```
ENABLE_CUBLAS_BACKEND:BOOL=OFF

//enable the cuFFT backend for DFT
ENABLE_CUFFT_BACKEND:BOOL=ON

ENABLE_CURAND_BACKEND:BOOL=OFF

ENABLE_CUSOLVER_BACKEND:BOOL=OFF
```
I will put a similar thing for the others.

Reply:
Updated in https://github.com/oneapi-src/oneMKL/pull/284/commits/42d77b3ec0e8937f3bb7b711ce85b0f9cd6c04d3